compute/rn/release-information/release-notes-31-01-update1.adoc

```diff
@@ -0,0 +1,99 @@
+:toc: macro
+== 31.01 Update 1 Release Notes
+
+The following table outlines the release particulars:
+
+[cols="1,4"]
+|===
+|Build
+|31.01.xx
+
+|Code name
+|Newton, 30.01 Update 1
+
+|Release date
+|September XX, 2023
+
+|Type
+|Maintenance Release
+
+|SHA-256
+|
+|===
+
+Review the https://docs.paloaltonetworks.com/prisma/prisma-cloud/31/prisma-cloud-compute-edition-admin/install/system_requirements[system requirements] to learn about the supported operating systems, hypervisors, runtimes, tools, and orchestrators.
+
+//You can download the release image from the Palo Alto Networks Customer Support Portal, or use a program or script (such as curl, wget) to download the release image directly from our CDN:
+
+// link
+
+toc::[]
+
+//[#cve-coverage-update]
+//=== CVE Coverage Update
+
+[#enhancements]
+=== Enhancements
+
+==== Severity Based Actions for Packages in Vulnerability Rules
+
+//CWP-48931
+==== Parallel Registry Scanning
+
+//CWP-41215
+==== Secrets Scanning with Agentless Security
+
+Agentless scanning now supports secret scanning across all workloads.
+Agentless scanning checks secrets in images, the container file systems, and the host file systems for AWS, Azure, GCP, and OCI.
+The following types of secrets are scanned.
+
+* Cloud keys
+* API tokens
+* Passwords
+* Configuration files
+* Private keys
+* Certificates
+
+
+[#new-features-core]
+=== New Features in Core
+
+
+//[#new-features-host-security]
+//=== New Features in Host Security
+
+//[#new-features-serverless]
+//=== New Features in Serverless
+
+//[#new-features-waas]
+//=== New Features in WAAS
+
+[#api-changes]
+=== API Changes and New APIs
+
+//[#breaking-api-changes]
+//=== Breaking Changes in API
+
+//[#deprecation-notice]
+//=== Deprecation Notice
+
+[#addressed-issues]
+=== Addressed Issues
+
+//CWP-35771
+* Fixed an issue where node count was missing for EKS clusters.
+
+//CWP-40710
+* Added missing value for `AccountIDs` macros in the vulnerabilities and compliance Webhook alert payloads sent to the alert notification providers.
+
+//CWP-50923
```

Comment:
@rcaballeromx Does this addressed issue belong here in "Newton Update 1" RN? As this was also listed as an addressed issue in "Newton" RN [here](https://docs.paloaltonetworks.com/prisma/prisma-cloud/31/prisma-cloud-compute-edition-release-notes/release-information/release-notes-31-00#:~:text=Fixed%20an%20issue%20where%20AKS%20cluster%20names)